README.md

```diff
@@ -1783,10 +1783,10 @@ RMB supports optimistic locking. By default it is disabled. Module developer can
 
```

Comment:
In line 1782: Can you also fix the typo "419 HTTP response code if 419" to become "409 HTTP response code if 409".